mp/src/game/shared/momentum/mom_gamemovement.cpp

```diff
@@ -1144,10 +1147,10 @@ bool CMomentumGameMovement::CheckJumpButton()
     mv->m_outWishVel.z += mv->m_vecVelocity[2] - startz;
     mv->m_outStepHeight += 0.1f;
 
-    if (g_pGameModeSystem->GameModeIs(GAMEMODE_RJ))
+    if (g_pGameModeSystem->GameModeIs(GAMEMODE_RJ) || g_pGameModeSystem->GameModeIs(GAMEMODE_SJ))
         mv->m_outStepHeight += 0.05f; // 0.15f total
 
-    if (!g_pGameModeSystem->GameModeIs(GAMEMODE_RJ))
+    if (!g_pGameModeSystem->GameModeIs(GAMEMODE_RJ) || g_pGameModeSystem->GameModeIs(GAMEMODE_SJ))
```

Comment:
You didn't negate the check for GAMEMODE_SJ here, is that intentional?

If not, you'll need to convert it to && as well (this code should only happen if not RJ and SJ)